src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+  font-weight: 600;
+  font-style: normal;
+}
+
+.big-lab,
+.small-lab {
+  display: block;
+  position: relative;
+  margin-bottom: 20px;
+}
+
+.big-lab::before {
+  content: "";
+  display: block;
+  background: url(images/Search.svg) no-repeat center;
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  height: 19px;
+  width: 19px;
+  background-size: cover;
+}
+
+.small-lab::before {
+  content: "";
+  position: absolute;
+  display: block;
+  background: url(images/Search.svg) no-repeat center;
+  left: 13px;
+  top: 15px;
+  height: 11px;
+  width: 11px;
+  background-size: cover;
+}
+
+.search-bar {
+  font-family: "Avenir", Arial;
+  color: #3d4e61;
+}
+
+.big {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 42px;
```

Comment:
add this to label

![image](https://user-images.githubusercontent.com/44294945/64124251-309bd280-cdaf-11e9-82e4-c20a37e3407c.png)

![image](https://user-images.githubusercontent.com/44294945/64124277-427d7580-cdaf-11e9-8470-a706894d1cf1.png)

Reply:
and use box-sizing

Reply:
> I don't understand what to addin label

add `height` to label
you need to fix element height

Reply:
can we speak tomorrow in the morning alone about this?